lib/langDialog/lang_dialog.dart

```diff
@@ -0,0 +1,189 @@
+import 'dart:core';
+
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:scroll_to_index/scroll_to_index.dart';
+
+class LangDialog extends StatefulWidget {
+  const LangDialog({Key? key}) : super(key: key);
+
+  @override
+  _LangDialogState createState() => _LangDialogState();
+}
+
+class _LangDialogState extends State<LangDialog> {
+  final AutoScrollController controller = AutoScrollController();
+
+  final List<String> _list = [
```

Comment:
use this 
[
    'Hindi',
    'Bengali',
    'Punjabi',
    'Kannada',
    'Tamil',
    'Telugu',
    'Marathi',
    'Urdu',
    'Malayalam',
    'Others'
  ];

Reply:
updated the list